src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/BlobProcessorTask.java

```diff
@@ -323,22 +324,38 @@ private void handleInvalidFileError(
             null
         );
 
-        ErrorMapping
-            .getFor(cause.getClass())
-            .ifPresent(errorCode -> {
-                try {
-                    sendErrorMessageToQueue(zipFilename, containerName, eventId, errorCode, cause);
-                } catch (Exception exc) {
-                    log.error(
-                        "Error sending notification to the queue."
-                            + "File name: " + zipFilename + " "
-                            + "Container: " + containerName,
-                        exc
-                    );
-                }
-            });
+        Optionals.ifPresentOrElse(
+            ErrorMapping.getFor(cause.getClass()),
+            (errorCode) -> trySendErrorMessageToQueueAndMoveFileToRejectedContainer(
+                zipFilename,
+                containerName,
+                cause, eventId,
+                leaseId,
+                errorCode
+            ),
+            () -> blobManager.tryMoveFileToRejectedContainer(zipFilename, containerName, leaseId)
+        );
```

Comment:
It should be more sth like this:
```
Optionals.ifPresentOrElse(
...
{
   send zip file
   move to rejected container when no error so far
},
() -> throw exception because we don't have necessary error code mapping (this is an existing bug)
```